app/src/main/java/com/example/tung/moviedb_07/utils/navigator/Navigator.java

```diff
@@ -1,10 +1,13 @@
 package com.example.tung.moviedb_07.utils.navigator;
 
 import android.app.Activity;
+import android.content.Context;
```

Comment:
ERROR: Unused import: android.content.Context. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports